### DISCUSSION TITLE
Actions not running

### DISCUSSION THREAD

**Discussion by svechinsky** [community/community#57444] [Actions]
### Select Topic Area

Bug

### Body

Is anyone else currently experiencing actions not triggering after a push?

**Comment by adamlui** [comment]
Yes @svechinsky I had to manually update 3 repo's, status.github.com doesn't reflect the down state so probably no one is working on fix, also this happens way too frequently for such a large corporation with so many servers repo's depend on >:(

**Comment by JoshMcCullough** [comment]
I added some new workflows and cannot get them to show up at all in the Actions tab or via `gh workflow list` ... perhaps related? In my case, these workflows are triggered by these events: `workflow_dispatch`, `scheduled`

There's no way for me to run them manually as they just are not "there".

**Reply by adamlui** [reply]
GitHub is so well-funded, it is unbelievable the frequency at which Actions goes down. Imagine all the times we didn't even realize it, how much code could've been affected...

**Reply by JoshMcCullough** [reply]
Sure, but it's is also an incredibly large and complex platform. Honestly I don't see issues often enough to complain. I just hope they fix this issue soon. :+1:

**Reply by adamlui** [reply]
That's no excuse, competitors with much less resources have equally large/complex platforms and don't face these issues. For example, PR's are also down right now, but virtually unheard of on those sites

**Reply by adamlui** [reply]
Also I use Actions up to hundreds of times a day is why I notice the frequency vs. you probably don't to feel it's somehow rare & not care

**Comment by johnDaniul** [comment]
We are also experiencing issues in Actions where scheduled actions are simply not queueing up at all.

**Comment by bahmanyaghoobi** [comment]
I am having the same problem. When I push to repo, actions don't get triggered.

**Comment by ChienKangLu** [comment]
Same here, I pushed code but nothing happens with the setup `on: [push, workflow_dispatch]` in workflow. By the way, six hours ago, everything is fine.

**Reply by ChienKangLu** [reply]
It works now :)

**Comment by NoNamer777** [comment]
I'm also experiencing issues with workflows not running on pushes to my repository

**Comment by TediPapajorgji** [comment]
Similar to this i've pushed up a new branch to a repo, and i'm unable to view the newly created branch in the github action:

<img width="325" alt="Screenshot 2023-06-07 at 1 15 27 PM" src="https://github.com/community/community/assets/5790075/bb95cad9-e60b-4375-95f6-78bd8517626a">

**Comment by theoturner** [comment]
Same problem here: actions that trigger on push are not running

**Comment by rea1shane** [comment]
Same problem, led me to push multiple nonsensical commits to verify this. SAD :(

**Comment by DeboBurro** [comment]
Our repository actions are getting triggered now.

**Comment by laurentiuspurba** [comment]
It's broken. I posted the same concern 2 weeks ago, and now, here we go again ...
